.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on: [push] # for testing
+  # push:
+  #   branches:
+  #     - main
+  # pull_request:
+  #   branches:
+  #     - main
+
+jobs:
+  build-and-test-python:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10", "3.11", "3.12"]
+    steps:
+      -  uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: "pip"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt -r dev-requirements.txt
```

Comment:
I think there is a more specific command expected

Reply:
This is the command to completely install all dependencies, I'm not aware of any other way to do this in our project setup. This is different from the full build command which is `pip install .`